### PR TITLE
Add rosdep key for python3-pyosmium

### PR DESCRIPTION
**Pull request by tdenewiler**

### CODE REVIEW

**Comment by tdenewiler** [comment]
Python interface for using Open Street Map files and data.

* Debian: https://packages.debian.org/search?suite=all&section=all&arch=any&searchon=names&keywords=python3-pyosmium
* Ubuntu: https://packages.ubuntu.com/search?keywords=python3-pyosmium&searchon=names&suite=all&section=all

File: rosdep/python.yaml [hunk 1]
```diff
@@ -6129,6 +6129,9 @@ python3-pymodbus:
   ubuntu:
     '*': [python3-pymodbus]
     xenial: null
+python3-pyosmium:
+  debian: [python3-pyosmium]
+  ubuntu: [python3-pyosmium]
```
Comment (jacobperron) [inline rosdep/python.yaml:6134]: Let's add a rule for Fedora as well: https://src.fedoraproject.org/rpms/pyosmium

**Review by jacobperron — COMMENTED** [review]
LGTM

One comment about adding an entry for Fedora as well.

File: rosdep/python.yaml (hunk 1 above)
Comment (tdenewiler) [reply rosdep/python.yaml:6136]: I added fedora to the Python2 key `pyosmium` as well. I am not as familiar with Fedora as I am with Debian/Ubuntu, but the link you provided looks like the package name is the same for Python 2/3 in Fedora.

Comment (jacobperron) [reply rosdep/python.yaml:6136]: Fedora 32 [dropped all Python 2 packages](https://fedoraproject.org/wiki/Changes/RetirePython2). I don't know if package `pyosmium` is compatible with Python 2 and Python 3, so maybe it's better to remove the fedora key for the Python 2 rosdep key.

@cottsay Maybe you'd like to chime in?

Comment (dirk-thomas) [reply rosdep/python.yaml:6136]: @cottsay Friendly ping.

Comment (hidmic) [reply rosdep/python.yaml:6136]: @cottsay friendly ping

Comment (cottsay) [reply rosdep/python.yaml:6136]: Sorry - I didn't get any notifications for this thread for some reason.

The source package is called `pyosmium`, but the python 3 subpackage is called [`python3-osmium`](https://src.fedoraproject.org/rpms/pyosmium/blob/master/f/pyosmium.spec#_31). There is no python 2 package in any currently supported Fedora platform.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -6129,6 +6130,10 @@ python3-pymodbus:
   ubuntu:
     '*': [python3-pymodbus]
     xenial: null
+python3-pyosmium:
+  debian: [python3-pyosmium]
+  fedora: [pyosmium]
```
Comment (cottsay) [inline rosdep/python.yaml:6135]:
```suggestion
  fedora: [python3-osmium]
```

File: rosdep/python.yaml [hunk 3]
```diff
@@ -332,6 +332,7 @@ pynput-pip:
       packages: [pynput]
 pyosmium:
   debian: [python-pyosmium]
+  fedora: [pyosmium]
```
Comment (cottsay) [inline rosdep/python.yaml:335]:
```suggestion
```

**Review by ivanpauno — APPROVED** [review]

**Review by cottsay — APPROVED** [review]

**Pull request merged**